go/libraries/doltcore/table/typed/parquet/reader.go

```diff
@@ -97,6 +99,35 @@ func (pr *ParquetReader) ReadRow(ctx context.Context) (row.Row, error) {
 	panic("deprecated")
 }
 
+// DECIMAL_BYTE_ARRAY_ToString converts a decimal byte array to a string
+// This is copied from https://github.com/xitongsys/parquet-go/blob/master/types/converter.go
+// while we wait for official release
+func DECIMAL_BYTE_ARRAY_ToString(dec []byte, prec int, scale int) string {
```

Comment:
(minor) I know the function name came from parquet-go, but I'd probably go ahead and change the name to match the naming/case style with the rest of the code. We could switch back to the exported function from parquet when that gets released, but it seems likely that we won't actually remember (or prioritize) to come back and do that clean up, since this function will be working just fine.